doc/rados/configuration/mclock-config-ref.rst

```diff
@@ -48,32 +77,74 @@ described in the :ref:`dmclock-qos` section.
-          "scrub", "snap trim", and "pg deletion") are given slightly lower
-          reservations, but higher weight and no limit. This ensures that
-          these operations are able to complete quickly if there are no other
-          competing services.
+mclock profiles can be broadly classified into *built-in* and *custom* profiles,
+
+Built-in Profiles
+-----------------
+Users can choose between the following built-in profile types:
+
+high_client_ops (*default*)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This profile allocates more reservation and limit to external-client ops
+as compared to background recoveries and other internal clients within
+Ceph. This profile is enabled by default. The table shows the resource
+controls parameters set by the profile:
+
++------------------------+-------------+--------+-------+
+|  Service Type          | Reservation | Weight | Limit |
++========================+=============+========+=======+
+| client                 | 50%         | 2      | MAX   |
++------------------------+-------------+--------+-------+
+| background recovery    | 25%         | 1      | 100%  |
++------------------------+-------------+--------+-------+
+| background best-effort | 25%         | 1      | MAX   |
++------------------------+-------------+--------+-------+
+
+high_recovery_ops
+^^^^^^^^^^^^^^^^^
+This profile allocates more reservation to background recoveries as
+compared to external clients and other internal clients within Ceph. For
+example, an admin may enable this profile temporarily to speed-up background
+recoveries during non-peak hours. The table shows the resource control
+parameters set by the profile:
+
++------------------------+-------------+--------+-------+
+|  Service Type          | Reservation | Weight | Limit |
++========================+=============+========+=======+
+| client                 | 30%         | 1      | 80%   |
++------------------------+-------------+--------+-------+
+| background recovery    | 60%         | 2      | 200%  |
++------------------------+-------------+--------+-------+
+| background best-effort | 1 (MIN)     | 1      | MAX   |
++------------------------+-------------+--------+-------+
+
+balanced
+^^^^^^^^
+This profile allocates equal reservation to client ops and background
+recovery ops.
+
++------------------------+-------------+--------+-------+
+|  Service Type          | Reservation | Weight | Limit |
++========================+=============+========+=======+
+| client                 | 40%         | 1      | 100%  |
++------------------------+-------------+--------+-------+
+| background recovery    | 40%         | 1      | 150%  |
++------------------------+-------------+--------+-------+
+| background best-effort | 20%         | 1      | MAX   |
++------------------------+-------------+--------+-------+
+
+.. note:: Across the built-in profiles, internal background best-effort clients
+          of mclock (for example "scrub", "snap trim", and "pg deletion") are
```

Comment:
let's remove "for example", there aren't more

Reply:
Done.